.devcontainer/docker-compose.override.yaml

```diff
@@ -30,3 +36,23 @@ services:
             - .venv/
         - path: lumigator/backend/pyproject.toml
           action: rebuild
+
+  mlflow:
```

Comment:
Should we bind mlflow so tightly to lumigator? Maybe we can just configure its url in lumigator and leave the user to decide how to start it up?

Reply:
I think it is probably the same situation as the rest of the tools. We rely on the tool to exist (e.g. Ray, Sql) and if they have their own, they can disable the usage. Once mlflow is implemented (the next PR that I'm working on), mlflow will be a mandatory part of the stack, similar to Ray. A user can make decisions about mlflow the same way they make decisions about Ray/Sql/Minio?

With this PR I put mlflow into the overrride (because the tracking service isn't actually being used anywhere so people not doing development dont need it yet) , but with the next PR I was planning on moving it into the main docker-compose file.